libraries/core_libs/network/src/tarcap/packets_handlers/votes_sync_packet_handler.cpp

```diff
@@ -24,6 +24,14 @@ void VotesSyncPacketHandler::validatePacketRlpFormat([[maybe_unused]] const Pack
 }
 
 void VotesSyncPacketHandler::process(const PacketData &packet_data, const std::shared_ptr<TaraxaPeer> &peer) {
+  if (!peer->votes_sync_requested_) {
+    LOG(log_er_) << "Received VotesSyncPacket from peer(" << packet_data.from_node_id_
+                 << ") we did not request. The peer may be a malicious player, will be disconnected";
+    disconnect(packet_data.from_node_id_, dev::p2p::UserReason);
```

Comment:
maybe we should use here `handleMaliciousSyncPeer` instead of just disconnect ?

Reply:
This method is from other packet handler, so I just added `set_peer_malicious` here and changed log